### DISCUSSION TITLE
Welcome to FXGraphics2D Discussions!

### DISCUSSION THREAD

**Discussion by jfree** [jfree/fxgraphics2d#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlexHomebrew** [comment]
Hello, I'm a long time Java developer (since 1.1) and am at a new company modifying legacy Java code.  I need to display some new charts of telemetry data and decided that jfreechart would be my charting package. The code is in a transition between AWT and JavaFX and I have a question: Does the Developer Guide have up to date instructions and examples of using jfreechart with JavaFX?
